src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -45,24 +46,23 @@ public GitHubPackage(IServiceProvider serviceProvider)
             this.serviceProvider = serviceProvider;
         }
 
-        protected override void Initialize()
+        protected override async tasks.Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
-            base.Initialize();
-            IncrementLaunchCount();
+            var usageTracker = await GetServiceAsync(typeof(IUsageTracker)) as IUsageTracker;
+            await tasks.Task.Run(() => usageTracker.IncrementLaunchCount());
```

Comment:
InitializeAsync would always be called in thread pool if package load is asynchronous, so call to usageTracker can be made directly as in both cases it will be executed in thread pool.